### PR TITLE
Sc94150/update schema

### PR DESCRIPTION
**Pull request by neo-anand-patel**
### Ticket

[sc94150]

### Changes made

- modify post request to send a pathName and displayName to remote endpoint
 
### Related PRs

- In the service

### Release Plan

- half of this ticket involves work on the service that saves this data when it is received in the remote endpoint. I am planning on finishing the other half of this ticket in [sc94150] in the service.
- The plan is to merge this PR into [sc95777] and then merge [sc95777] from this repository and then [sc94150] in the service

### Checklist

- [x] Have you fully tested your PR locally

### CODE REVIEW

**Comment by neo-anand-patel** [comment]
Old PR: https://github.com/neofinancial/coverage-upload-action/pull/20

**Review by PeterHuang516 — APPROVED** [review]

**Review by leah-is-offline — CHANGES_REQUESTED** [review]
changelog date hahaha

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Coverage Upload Action Changelog
 
+## 2.1.0 (April 2, 2022)
```
Comment (leah-is-offline) [inline CHANGELOG.md:3]: its June

**Review by leah-is-offline — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (neo-anand-patel) [reply CHANGELOG.md]: That is unfortunate, thanks lol

**Pull request merged**